packages/core/src/decorators/Transactional.ts

```diff
@@ -0,0 +1,35 @@
+import type { TransactionOptions } from '../enums';
+import type { AsyncFunction, GetContext } from '../typings';
+import { RequestContext } from '../utils/RequestContext';
+import { TransactionContext } from '../utils/TransactionContext';
+import { resolveGetContext } from '../utils/Utils';
+
+type TransactionalOptions<T> = TransactionOptions & { getContext?: GetContext<T> };
+
+/**
+ * you can specify context to open transaction by providing `getContext` property.
```

Comment:
lets make this more informative, starting a comment with "you can" feels weird. describe what the decorator is about, not just one of its options.

```suggestion
 * you can specify context to open transaction by providing `getContext` option.
```